operator/tests/central/basic/74-errors.yaml

```diff
@@ -1,9 +0,0 @@
-apiVersion: v1
-kind: PersistentVolumeClaim
-metadata:
-  name: central-db
```

Comment:
Shouldn't we add these to the assert file now to make sure these are created?

Reply:
It's there but you are right, it makes sense check for stackrox-db PVC as well